seagull/rules.py

```diff
@@ -48,6 +48,55 @@ def life_rule(X: np.ndarray, rulestring: str) -> np.ndarray:
     return birth_rule | survival_rule
 
 
+def rps_life_rule(X: np.ndarray, cycle_thresholds: list) -> np.ndarray:
+    """A generalized rock-paper-scissors (RPS) life rule that accepts a rulestring in B/S notation
```

Comment:
Let's limit the scope for now and don't include rulestrings for rock-paper-scissors

```suggestion
    """A generalized rock-paper-scissors (RPS) life rule
```

Reply:
Funnily enough I think was just a brainfart/typo on my part. I agree.